tango/__main__.py

```diff
@@ -164,22 +160,14 @@ def main(
     ctx,
     config: Optional[str],
     log_level: Optional[str],
-    no_logging: bool,
     file_friendly_logging: bool = False,
 ):
     config: TangoGlobalSettings = TangoGlobalSettings.find_or_default(config)
 
-    if no_logging or config.no_logging:
-        config.no_logging = True
-        log_level = None
-        config.log_level = None
-    elif log_level is not None:
-        config.log_level = log_level
-    else:
-        log_level = config.log_level
+    config.log_level = log_level
```

Comment:
```suggestion
    if log_level is not None:
        config.log_level = log_level
```

Reply:
That doesn't work though. If `log_level` is `None`, we need it to be `None` in `initialize_logging`, so that it can read out the environment variable. I think we need a different solution altogether.

Reply:
I put a different solution into 03049fa.

Reply:
The point of the config was to control the log level (among other things). With https://github.com/allenai/tango/commit/03049fa37c99a5da8b182ba77cb41ff6b6986673 you are just **setting** the config's log level based on the environment variable or the command line argument.

Reply:
What's the difference between "control the log level" and "set the log level"?